src/emacs.ts

```diff
@@ -19,6 +19,17 @@ export function activate(context: vscode.ExtensionContext): void {
         removeSelection();
         inMarkMode = false;
     }));
+    
+    context.subscriptions.push(vscode.commands.registerCommand('emacs.cutToEndOfLine', (context) => {
```

Comment:
This doesn't work as intended when the text between the cursor and eol matches text elsewhere in the document. Need to tweak.